src/app/business/block/bucket-detail/bucket-detail.component.ts

```diff
@@ -101,6 +105,76 @@ export class BucketDetailComponent implements OnInit {
       this.getTypes();
     }
     );
+    this.copySelectedDir = window.sessionStorage['searchIndex'] != "" ? JSON.parse(window.sessionStorage.getItem("searchIndex")) : [];
+  }
+  clickOperate(){
+    if(this.selectedDir.length >0){
+      this.isReadyCopy = false;
+      this.copySelectedDir = lodash.cloneDeep(this.selectedDir);
+    }else{
+      this.isReadyCopy = true;
+      this.copySelectedDir = window.sessionStorage['searchIndex'] != "" ? JSON.parse(window.sessionStorage.getItem("searchIndex")) : [];
+      this.isReadyPast = this.copySelectedDir.length != 0 ? false :true;
+    }
+    this.moreItems = [
+      {
+        disabled: this.isReadyCopy,
+        label: 'copy', command: () => {
+          this.copyObject();
+        }
+      },
+      {
+        disabled: this.isReadyPast,
+        label: 'paste', command: (y) => {
```

Comment:
Use title case.
paste -> Paste
is it possible to add icons for copy / paste?

Reply:
There are no icons on aws, so we didn't add

Reply:
This optimization will be included in the submission of the next PR, because I have another local requirement to submit PR, which is also the requirement of the bucket.

Reply:
Ok.